.travis.yml

```diff
@@ -33,7 +37,7 @@ branches:
     - master
 
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c "ALTER USER travis WITH PASSWORD 'travis';"
```

Comment:
Is there a reason we need to set the password?

Reply:
Yes, if you look in [the Travis log](https://api.travis-ci.org/v3/job/374023986/log.txt) you can see that they install Postgres 10 with `initdb --auth-host md5`, which [requires password login for users](https://www.postgresql.org/docs/10/static/auth-methods.html#AUTH-PASSWORD). We could optionally create our own new user and set its password but I thought it less complex to just give the `travis` user a password to use. This method is also suggested in [this discussion](https://github.com/travis-ci/travis-ci/issues/8537#issuecomment-383758631).